tests/unit/models/addon-test.js

```diff
@@ -757,19 +773,25 @@ describe('models/addon.js', function() {
       }
     });
 
-    it('should move files in the root of the addons app/styles tree into the app/styles path', function() {
+    it('should move files in the root of the addons app/styles tree', function() {
       builder = new broccoli.Builder(addon.treeFor('styles'));
 
       return builder.build()
         .then(function(results) {
           let outputPath = results.directory;
 
           let expected = [
-            'app/',
-            'app/styles/',
-            'app/styles/foo-bar.css',
+            'foo-bar.css',
```

Comment:
It looks that some of the existing tests (like this line) have changed and I found the same situation in other tests.

I don't know much about the internal of the ember-cli project, but I think the `MU addon style` feature should not affect Classic addons or apps, hence I am not sure if it is convenient to change the existing behaviour of these tests when running without enabling `MODULE_UNIFICATION`.